keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -802,14 +824,18 @@ the next question. -->
 - [x] Metrics
   - Metric name: `aggregator_discovery_aggregation_duration`
   - Components exposing the metric: `kube-server`
-  - This is a metric for exposing the time it took to aggregate all the
+  - This is a metric for exposing the time it took to aggregate all the api resources.
```

Comment:
I find it surprising that a duration counter like this wouldn't have a "count" value associated with it?

https://prometheus.io/docs/concepts/metric_types/

Both `Histogram` and `Summary` have a corresponding `_count` metric?

Reply:
The count metric is listed below `aggregator_discovery_aggregation_count`. Are you suggesting that we should include additional sum and bucket metrics to be able to form a histogram?

Reply:
No, I guess I was just thinking that the `_count` metric is added automatically, right?